src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1872,7 +1591,7 @@ static void processEntity(ContextRegistrationResponse* crr, EntityIdVector enV,
 *
 * processAttribute -
 */
-static void processAttribute(ContextRegistrationResponse* crr, AttributeList attrL, BSONObj attribute)
+static void processAttribute(ContextRegistrationResponse* crr, const AttributeList& attrL, BSONObj attribute)
```

Comment:
Rigth... I focused on `attrL` when chanching to `const &` but of course it can also be used in the others arguments in the signature (in this case, the `attribute`). I'll do it.